README.md

```diff
@@ -0,0 +1,29 @@
+# Team Up Server API
+
+Simple node + express server implementing a REST API
+
+## Technologies
+
+- Node.js
+- Express
+- JWT Authentication
+
+### Version
+
+0.1.0
+
+## Usage
+
+### Installation
+
+Install the dependencies
+
+```sh
+$ npm install
+```
+
+Run server
```

Comment:
Please specify the production and dev environment...

For production `npm start` will do the trick but for development you surely will prefer the `npm run watch`